https://github.com/shlomif/fortune-mod/pull/69

The intention of LOCALDIR seems to be to allow local cookies to be
provided by the sysadmin in e.g. /usr/local/share/fortune,
but if set, before this patch, even cookies distributed
by upstream were stored there (and nothing in COOKIEDIR, e.g.
/usr/share/fortune - https://bugs.gentoo.org/865809).

Originally, we were setting LOCALDIR=COOKIEDIR=/usr/share/fortune to
workaround the issue, but see https://bugs.gentoo.org/857246.
--- a/datfiles/CMakeLists.txt
+++ b/datfiles/CMakeLists.txt
@@ -68,7 +68,7 @@ ADD_CUSTOM_TARGET(
 )
 INSTALL(
     FILES ${_install_unrot_cookies}
-    DESTINATION "${LOCALDIR}"
+    DESTINATION "${COOKIEDIR}"
 )
 
 IF (NOT NO_OFFENSIVE)
--- a/datfiles/off/CMakeLists.txt
+++ b/datfiles/off/CMakeLists.txt
@@ -54,5 +54,5 @@ ADD_CUSTOM_TARGET(
 )
 INSTALL(
     FILES ${_install_rotated_cookies}
-    DESTINATION "${LOCALODIR}"
+    DESTINATION "${OCOOKIEDIR}"
 )
